src/ios/CDVInAppBrowser.m

```diff
@@ -937,6 +952,25 @@ - (void)webView:(UIWebView*)theWebView didFailLoadWithError:(NSError*)error
     [self.navigationDelegate webView:theWebView didFailLoadWithError:error];
 }
 
+#pragma mark NSURLConnectionDataDelegate
+
+- (void)connection:(NSURLConnection *)connection willSendRequestForAuthenticationChallenge:(NSURLAuthenticationChallenge *)challenge {
+    // Allow self-signed SSL certificates from URLs that have the same host as the base URL.
```

Comment:
Is this limited to self signed certificates, or are we allowing any invalid certificates when we disable checking?